README.md

```diff
@@ -27,7 +26,7 @@ on installing `git` for Linux and Mac.
 libvirt to accept TCP connections for remote installations. Ensure firewall rules allow remote connections to the host.
```

Comment:
Possible to add a link describing how to do these ?

Reply:
I will create another doc for host setup uder docs named host-setup and will link it here in anothe PR.